go.mod

```diff
@@ -63,7 +64,8 @@ require (
 	github.com/lestrrat-go/httpcc v1.0.1 // indirect
 	github.com/lestrrat-go/httprc v1.0.4 // indirect
 	github.com/lestrrat-go/iter v1.0.2 // indirect
-	github.com/lestrrat-go/jwx v1.2.23 // indirect
+	github.com/lestrrat-go/jwx v1.2.25 // indirect
+	github.com/lestrrat-go/jwx/v2 v2.0.6 // indirect
```

Comment:
Within services, we should only use V2 ideally, we need to evaluate why v1 is still needed here??

Reply:
Corrected all dependencies, now jwx only uses version v2, v2.0.8